convert/src/label_studio_to_badgerdoc/badgerdoc_format/badgerdoc_format.py

```diff
@@ -49,13 +51,22 @@ def convert_from_labelstudio(self, labelstudio_data: LabelStudioModel):
     def convert_from_text(self, text: str):
         self.tokens_page = self.text_converter.convert(text)
 
-    def export_tokens(self, path: Path) -> None:
+    def convert_from_pdf(self, pdf):
+        self.tokens_page = self.pdf_converter.convert(pdf)
+
+    def export_tokens(self, tokens_path: Path) -> None:
         if self.tokens_page:
-            path.write_text(self.tokens_page.json(indent=4, by_alias=True))
+            if isinstance(self.tokens_page, list):
+                for i, part in enumerate(self.tokens_page, start=1):
+                    path_file = tokens_path / Path(f"{i}.json")
+                    path_file.write_text(part.json(by_alias=True))
+            else:
+                path_file = tokens_path / Path("1.json")
+                path_file.write_text(self.tokens_page.json(by_alias=True))
```

Comment:
I don't understand here what other types of self.tokens_page we can get rather than list.

Reply:
from this fucntion: 
```
def convert_from_text(self, text: str):
        self.tokens_page = self.text_converter.convert(text)
```